org.adempiere.base/src/org/compiere/model/MColumn.java

```diff
@@ -750,7 +748,21 @@ public String getConstraint(String tableName)
 		}
 		return "";
 	}	//	getConstraint
-	
+
+	/**
+	 * Get a correct Unique UU index name
+	 * @param columnName
+	 * @param indexName
+	 * @return validated indexName
+	 */
+	public static String validateUuIdxIndex(String columnName, String indexName) {
```

Comment:
a better name for this method is maybe getUUIDIndexName.

Also, I think the method must just receive the tableName as parameter.

Reply:
Hi,
I can change the method name and its parameter, but in this case, I should also move it to MTable or MTableIndex?
wdyt?

Reply:
Done